src/helpers/error.rs

```diff
@@ -16,6 +20,11 @@ pub enum Error {
         #[source]
         inner: BoxError,
     },
+    #[error("An error occurred while sending data to unknown helper")]
```

Comment:
Lets maybe not intrigue the readers of this error message - "unknown helper" sparks interest as I'd love to know what helper was destined to receive a message from us but I won't be able to :)

this problem of actually adding *context* to error messages is a bit tricky to address in Rust. I don't want to solve it in this PR, but we should keep track of it, so maybe create an issue for it.

Reply:
haha i certainly don't want to intrigue the readers. here's an issue to add context to this error specifically, and more broadly to have a way to add context: https://github.com/private-attribution/ipa/issues/226